applications/mne_x/plugins/eegosports/eegosports.cpp

```diff
@@ -567,13 +592,16 @@ void EEGoSports::run()
 void EEGoSports::showSetupProjectDialog()
 {
     // Open setup project widget
-    if(m_pEEGoSportsSetupProjectWidget == NULL)
+    if(m_pEEGoSportsSetupProjectWidget == NULL) {
```

Comment:
in future we should consider assigning nullptr to an empty pointer, and cheking also for a nullptr. NULL is not optimal since NULL could also be an address.